solr/core/src/java/org/apache/solr/handler/component/SearchHandler.java

```diff
@@ -289,6 +295,19 @@ public void handleRequestBody(SolrQueryRequest req, SolrQueryResponse rsp) throw
       rb.requestInfo.setResponseBuilder(rb);
     }
 
+    //TODO: Should this be for indexing requests as well?
+    CircuitBreakerManager circuitBreakerManager = req.getCore().getCircuitBreakerManager();
+    Map<CircuitBreakerType, CircuitBreaker> trippedCircuitBreakers = circuitBreakerManager.checkAllCircuitBreakers();
```

Comment:
Do we want to time this and include that in the response (useful for both passed and failed breakers)

Reply:
Not sure if I understood. Would you mean the epoch timestamp when the circuit breaker triggered?

Reply:
I mean in response we send timing info with `debug=true`(see `RTimerTree subt = timer.sub( "prepare" );`), maybe this is worthwhile to include there as well since otherwise the times may not add up

Reply:
Added, thanks